README.md

```diff
@@ -34,7 +35,7 @@
 </ul>
 <h2 id="how-emdaer-works">How emdaer works</h2>
 <p>emdaer processes template files and writes the resulting files to your project.</p>
-<p>We match <code>.emdaer/(**/*).emdaer(.md)</code> and use the captured part of each matched file to determine the path for the output.</p>
+<p>We match `.emdaer/(*_/_).emdaer(.md)` and use the captured part of each matched file to determine the path for the output.</p>
```

Comment:
Is this a prettier bug?

Reply:
kind of seems like a bug ya, though. i'm also interested in why those code tags went away